go.mod

```diff
@@ -1,17 +1,24 @@
 module github.com/openshift/ofcir
 
-go 1.20
+go 1.23
```

Comment:
@danmanor it looks like this impacted the [e2e-test job](https://github.com/openshift/appliance/blob/605d8a1019a9e232fc2e511b2d47e91c2adec758/data/services/install/start-local-registry-upgrade%40.service#L10)

Reply:
checking now. Anyway, this change is essential as this is the actual go version we use in the image

Reply:
@andfasano Can you please approve ? https://github.com/openshift/release/pull/65269

Reply:
We should probably move this logic to ofcir repo btw to avoid this